pygame_code_examples/move_keyboard.py

```diff
@@ -66,11 +66,7 @@ def draw_stick_figure(screen, x, y):
         elif event.type == pygame.KEYDOWN:
             # Figure out if it was an arrow key. If so
             # adjust speed.
-            if event.key == pygame.K_LEFT:
```

Comment:
I think we should just leave the examples as they were. 
A bonus challenge would be to remove this file from the commit using only git commands.